src/main/webapp/index.html

```diff
@@ -4,4 +4,16 @@
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
 <script src="script.js"></script>
-<p>Hello world!</p>
+<form name="lectureForm" action="/lecture" method="POST" onsubmit="return validateForm()" required>
+  <h2>Add a new lecture!</h2>
+  <h3>Enter a name for this lecture:</h3>
+  <input type="text" placeholder="Lecture Name" name="name-input" />
+  <br />
+  <h3>Enter the YouTube link for this lecture:</h3>
+  <input type="text" placeholder="Lecture URL" name="link-input" />
```

Comment:
This can be `type="url"` : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

Reply:
fixed